src/sedcli_main.c

```diff
@@ -78,6 +86,17 @@ static cli_option setpw_opts[] = {
 };
 
 static cli_command sedcli_commands[] = {
+	{
+		.name = "sed-discovery",
```

Comment:
To be consistent with other command definition can you please use just "discovery"  w/o any prefixes.